dashboarddoc.md

```diff
@@ -0,0 +1,156 @@
+   </div>
+
+   - *Weather station site creation form*
+   <img class="my-3" src="/images/SiteForm.png"/>
+
+   - *Power plant site creation form*
+   <img class="my-3" src="/images/plant_site_form.png"/>
+
+3. After submission, you will be redirected to a Site
+   page which displays the new Site’s metadata and allows you
+   to create associated Observations and Forecasts. See
+   [Create an Observation or Forecast](#create-an-observation-or-forecast).
+
+   *Power Plant Site Page*
+   <img class="my-3" src="/images/plant_site.png"/>
+
+
+
+Create an Observation or Forecast
+---------------------------------
+{: .anchor}
+
+To create an Observation or Forecast, an associated site must already exist.
+See [Create a Site](#create-a-site) for instructions.
+
+### Create an Observation
+{: anchor} 
+
+1.  Navigate to the Site listing page using the ‘Sites’ link in the left
+	sidebar. Select the site for which you are adding an Observation.
+
+2.  Click the “Create new Observation” button on the Site page.
+  <img class="my-3" src="/images/weather_site.png"/>
+
+3.  Enter metadata for your Observation. On submission, you will be redirected
+    to an Observation page which displays the new Observation metadata and a
+    link to add Observation data. See [Upload Data](#upload-observation-data).
+- *Observation form*
+<img class="my-3" src="/images/weather_obs.png"/>
+- *Created Observation page*
+<img class="my-3" src="/images/observation.png"/>
+
+### Create a Forecast
+{: anchor}
+
+1.  Navigate to the Site listing page using the ‘Sites’ link in the left
+	sidebar. Select the site for which you are adding a Forecast.
+
+2.   Click the “Create new Forecast” button on the Site page.
+  <img class="my-3" src="/images/plant_site.png"/>
+
+3.  Enter metadata for your Observation. On submission, you will be redirected
+    to an Observation page which displays the new Forecast metadata and a
+    link to add Forecast data. See [Upload Data](#upload-forecast-data).
+- *Forecast form*
+<img class="my-3" src="/images/forecast_form.png"/>
+- *Created Forecast page*
+<img class="my-e" src="/images/forecast.png"/>
+
+
+Upload data
+-----------
+{: .anchor}
+
+To upload data, an associated Site and Observation or Forecast object
+must already exist. See [Create a Site](#create-a-site) or
+[Create an Observation or Forecast](#create-an-observation-or-forecast)
+for instructions. The instructions here will describe the process of
+uploading data using the dashboard.
+Uploading data may be automated using the API, see
+[https://dev-api.solarforecastarbiter.org/](https://dev-api.solarforecastarbiter.org/)
+for detailed documentation.
+
+### Upload Observation Data
+{: anchor}
+
+1.  From Sites listing page, click on a site.
+
+2.  Click ‘Observations’ to find the Observations listing for that site.
+
+3.  Select an Observation. Click ‘Upload Data’ on the Observation page.
+<img class="my-3" src="/images/observation.png"/>
```

Comment:
the "M" in the units string should be "m", but probably not worth the effort to change right now.

Reply:
interval label should be "beginning" here and in the next image. probably is worth changing that one, so might as well do the same for the M above.